packages/react-components/src/components/ParticipantsButton.tsx

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  ContextualMenuItemType,
+  DefaultButton,
+  IButtonProps,
+  IContextualMenuItem,
+  IContextualMenuProps,
+  IStyle,
+  Label,
+  Stack,
+  concatStyleSets,
+  mergeStyles
+} from '@fluentui/react';
+import { UserFriendsIcon } from '@fluentui/react-northstar';
+import copy from 'copy-to-clipboard';
+import React, { useCallback, useMemo } from 'react';
+import { ParticipantList, ParticipantListProps } from './ParticipantList';
+import {
+  controlButtonLabelStyles,
+  controlButtonStyles,
+  defaultParticipantListContainerStyle
+} from './styles/ControlBar.styles';
+import { BaseCustomStylesProps } from '../types';
+
+/**
+ * Styles Props for ParticipantsButton component
+ */
+export interface ParticipantsButtonStylesProps extends BaseCustomStylesProps {
+  /** Styles of ParticipantList container */
+  participantListContainerStyle?: IStyle;
+}
+
+/**
+ * Props for ParticipantsButton component
+ */
+export interface ParticipantsButtonProps extends IButtonProps {
+  /**
+   * Whether the label is displayed or not.
+   *
+   * @default false
+   *
+   */
+  showLabel?: boolean;
+  /**
+   * Props of ParticipantList component
+   */
+  participantListProps: ParticipantListProps;
+  /**
+   * Allows users to pass an object containing custom CSS styles.
+   * @Example
+   * ```
+   * <MessageThread styles={{ root: { background: 'blue' } }} />
+   * ```
+   */
+  styles?: ParticipantsButtonStylesProps;
+  /**
+   * URL to invite new participants to the current call
+   */
+  callInvitationURL?: string;
+  /**
+   * CallBack to mute all remote participants
+   */
+  onMuteAll?: () => void;
+}
+
+/**
+ * `ParticipantsButton` allows you to easily create a component rendering a participants button. It can be used in your ControlBar component for example.
+ * This button, by default, contains dropdown menu items defined through its property `menuProps` (user presence, number of remote participants with a list
+ * as sub-menu and a copy-to-clipboard button of the call invite URL).
+ * This `menuProps` property is of type [IContextualMenuProps](https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu#IContextualMenuProps).
+ *
+ * @param props - of type ParticipantsButtonProps
+ */
+export const ParticipantsButton = (props: ParticipantsButtonProps): JSX.Element => {
+  const {
+    participantListProps,
+    showLabel = false,
+    callInvitationURL,
+    styles,
+    onMuteAll,
+    onRenderIcon,
+    onRenderText
+  } = props;
+
+  participantListProps.excludeMe = true;
```

Comment:
I'd add a comment that we show local user in the context menu and so skip from participant list.